libswirl/hw/maple/maple_cfg.cpp

```diff
@@ -124,8 +124,7 @@ bool maple_atomiswave_coin_chute(int slot)
 
 void mcfg_Create(MapleDeviceType type, u32 bus, u32 port, s32 player_num = -1)
 {
-	if (MapleDevices[bus][port] != NULL)
-		delete MapleDevices[bus][port];
+	delete MapleDevices[bus][port];
```

Comment:
The null check here is intentional - there's no guarantee that the device has been instantiated before